pyuvsim/simsetup.py

```diff
@@ -653,7 +653,8 @@ def parse_telescope_params(tele_params, config_path=''):
     return_dict['antenna_names'] = np.array(antnames.tolist())
     return_dict['antenna_numbers'] = np.array(antnums)
     antpos_enu = np.vstack((E, N, U)).T
-    return_dict['antenna_positions'] = uvutils.ECEF_from_ENU(antpos_enu, *telescope_location) - tele_params['telescope_location']
+    return_dict['antenna_positions'] = uvutils.ECEF_from_ENU(antpos_enu, *telescope_location) - tele_params[
+        'telescope_location']
```

Comment:
I fully understand breaking lines for ease of reading but breaking the line on the argument to `tele_params` I actually find more confusing. Can we break before/after the operator? I know PEP has slightly conflicting ideas about this. https://lintlyci.github.io/Flake8Rules/rules/W503.html

current best practice might be break before the operator? 

This same thing happens a few times actually.

Reply:
In pyuvdata, we've chose to break before the operator. We enforce this with our choice of ignore arguments to pydocstyle: `--ignore=E501,W503`

Can we do a similar thing for flake8? These conventions have been thoroughly discussed by the pyuvdata developers and agreed on there.

Reply:
Yup, so all of these (this and below) were auto-formatted by my editor, and I agree with you guys. I'll fix these up.

Reply:
Fixed in b7e89d70fab47c00273dbfa95fa5f5c82f3af8b0